raphtory-graphql/schema.graphql

```diff
@@ -1042,6 +1222,52 @@ input InputEdge {
 }
 
 
+"""
+Provides access to the intervals between temporal entries of an object.
+"""
+type Intervals {
+	"""
+	List time intervals between consecutive timestamps in milliseconds.
+	"""
+	list: [Int!]!
+	"""
+	List millisecond time intervals between consecutive timestamps in reverse order.
+	"""
+	listRev: [Int!]!
+	"""
+	Fetch one page of intervals between consecutive timestamps with a number of items up to a specified limit,
+	optionally offset by a specified amount. The page_index sets the number of pages to skip (defaults to 0).
+	
+	For example, if page(5, 2, 1) is called, a page with 5 items, offset by 11 items (2 pages of 5 + 1),
+	will be returned.
+	"""
+	page(limit: Int!, offset: Int, pageIndex: Int): [Int!]!
+	"""
+	Fetch one page of intervals between consecutive timestamps in reverse order with a number of items up to a specified limit,
+	optionally offset by a specified amount. The page_index sets the number of pages to skip (defaults to 0).
+	
+	For example, if page(5, 2, 1) is called, a page with 5 items, offset by 11 items (2 pages of 5 + 1),
+	will be returned.
+	"""
+	pageRev(limit: Int!, offset: Int, pageIndex: Int): [Int!]!
+	"""
+	Compute the mean interval between consecutive timestamps. Returns None if fewer than 1 timestamp.
+	"""
+	mean: Float
+	"""
+	Compute the median interval between consecutive timestamps. Returns None if fewer than 1 timestamp.
+	"""
+	median: Int
+	"""
+	Compute the maximum interval between consecutive timestamps. Returns None if fewer than 1 timestamp.
+	"""
+	max: Int
+	"""
+	Compute the minimum interval between consecutive timestamps. Returns None if fewer than 1 timestamp.
+	"""
+	min: Int
+}
```

Comment:
Putting a comment here, but it about the below 

Should the mutableGraph mutableNode and mutableEdge not be updated with the new time type?